consolelabs/mochi-discord
# feat: invest command

**What does this PR do?**

-  [x] Invest homepage
-  [x] Invest by token

**How to test**

-   Type `/invest` to see all earning options 
-  Type `/invest eth` to see all earning options for *ETH*

**Media (Loom or gif)**

![Screenshot 2023-06-21 at 15 48 31](https://github.com/consolelabs/mochi-discord/assets/11255278/bfab964e-121c-4ec9-8a56-690c767f5d1a)

![Screenshot 2023-06-21 at 15 31 17](https://github.com/consolelabs/mochi-discord/assets/11255278/fe691ef6-54d7-4685-9072-773fdf8365aa)
